emcc.py

```diff
@@ -2771,14 +2771,19 @@ def modularize():
       # after document.currentScript is gone, so we save it.
       # (when MODULARIZE_INSTANCE, an instance is created
       # immediately anyhow, like in non-modularize mode)
+      if shared.Settings.EXPORT_ES6:
+        GETSCRIPT = "performance.getEntries().slice(-1)[0].name"
```

Comment:
If this returns the current script's path, cool, good to know there's another method, but wouldn't `import.meta.url` be simpler?

Reply:
Huh, I didn't know that was a thing. Yeah that's much better.